src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -670,13 +681,28 @@ def get_guardian_duplication_row_counts(
             )
         ]
     )
+    metadata_df["group_rows.row_probability_in_college_group_quarters_under_24"] = len(
+        df.loc[(df["age"] < 24) & (df["housing_type"] == "College")]
+    )
 
     return metadata_df
 
 
 def _get_metadata_counts(
     observer: str, df: pd.DataFrame, location: str, year: int
 ) -> pd.DataFrame:
+    # FIXME: import this from pseudopeople after release
+    COPY_HOUSEHOLD_MEMBER_COLS = {
```

Comment:
We currently can't important this from pseudopeople because pseudopeople will be on a release branch and not main.

Reply:
Are you saying that this object isn't yet released in pseudopeople and so you need to wait for v1.0?

Reply:
Yea. I could import it using editable install but the github builds will still break.